project/Build.scala

```diff
@@ -93,7 +93,7 @@ object Scrooge extends Build {
     libraryDependencies ++= Seq(
       "org.scalatest" %% "scalatest" % "2.2.4" % "test",
       "org.scalacheck" %% "scalacheck" % "1.12.2" % "test",
-      "junit" % "junit" % "4.10" % "test" exclude("org.mockito", "mockito-all")
+      "junit" % "junit" % "4.12" % "test"
```

Comment:
Junit doesn't ship with mockito, so we don't ned to exclude it.
I bumped it to 4.12 just to stay up-to-date (version 4.10 is from 29-Sep-2011, version 4.12 is from 04-Dec-2014)
I'm not sure if this builds with 4.10 or not, I didn't really test. Do you want me to change it back to 4.10?